programs/steward/src/state/steward_state.rs

```diff
@@ -236,7 +230,37 @@ impl IdlBuild for StewardStateEnum {
     }
 }
 
+// BITS 0-7 COMPLETED PROGRESS FLAGS
+pub const COMPUTE_SCORE: u32 = 1 << 0;
+pub const COMPUTE_DELEGATIONS: u32 = 1 << 1;
+pub const EPOCH_MAINTENANCE: u32 = 1 << 2;
+pub const PRE_LOOP_IDLE: u32 = 1 << 3;
+pub const COMPUTE_INSTANT_UNSTAKES: u32 = 1 << 4;
+pub const REBALANCE: u32 = 1 << 5;
+pub const POST_LOOP_IDLE: u32 = 1 << 6;
+// BITS 8-15 RESERVED FOR FUTURE USE
+// BITS 16-23 OPERATIONAL FLAGS
+pub const CHECKED_VALIDATORS_REMOVED_FROM_LIST: u32 = 1 << 16;
+pub const RESET_TO_IDLE: u32 = 1 << 17;
```

Comment:
could we make this an enum so the other files don't have to import all these constants?
```rust
pub enum StatusFlags {
  ComputeScore = COMPUTE_SCORE,
  ComputeDelegations = ...
}

Reply:
It all started out with an enum - but I fought Rust allowing me to go between u32 and the enum. I had all the functions and everything compiling, then I decided it was overly verbose for something that should be simple. That being said, I can change it back into an enum if you'd like.